projects/core/src/features-config/feature-toggles/config/feature-toggles.ts

```diff
@@ -507,6 +507,12 @@ export interface FeatureTogglesInterface {
    * customization buttons in the BottomHeaderSlot in SmartEdit.
    */
   cmsBottomHeaderSlotUsingFlexStyles?: boolean;
+
+  /**
+   * When enabled, a new theme switcher will be used.
+   * It supports default theme together with High Contrast Dark and High Contrast Light.
```

Comment:
It's a "breaking change toggle", so let's be empathetic and help customers understand the scope of the breaking changes, so they can estimate the amount of time required for them to adapt to this breaking change.

```suggestion
   * 1. It uses the new `SiteThemeService` as the source of truth for the "site theme" value
   * (this value can change over time, e.g. when selecting new value in the new `ThemeSwitcherComponent`).
   * Previously the "site theme" could be set only on the page start (via the static config `config.context.theme` or via CMS, when using the feature of the "automatic site-context configuration").
   * 2. Now, when no custom theme is selected, the default theme value is an empty string `''`,
   * unless you configure it differently via the global config `config.context.theme` (or via CMS).
   * Previously, there the non-defined theme had a value `undefined`.
```